src/js/tests/hooks/useWizard.test.jsx

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+
+import { act, renderHook } from '@testing-library/react-hooks';
+
+import useWizard from 'hooks/useWizard';
+
+describe('Changing steps', () => {
+  const steps = [
+    {
+      key: 1,
+      Component: () => (<div>First compoment</div>),
+      title: 'First',
+    },
+    {
+      key: 2,
+      Component: () => (<div>Second compoment</div>),
+      title: 'Second',
+    },
+    {
+      key: 3,
+      Component: () => (<div>Third compoment</div>),
+      title: 'Third',
+    },
+    {
+      key: 4,
+      Component: () => (<div>Fourth compoment</div>),
+      title: 'Fourth',
+    },
+  ];
+
+  it('wizard will start with the initial key', () => {
+    const { result } = renderHook(() => useWizard({ initialKey: 3, steps }));
+
+    expect(result.current[0].key).toEqual(3);
+  });
+
+  it('wizard goes to next step', () => {
+    const { result } = renderHook(() => useWizard({ initialKey: 2, steps }));
+
+    act(() => {
+      result.current[1].next();
+    });
+
+    expect(result.current[0].key).toEqual(3);
+  });
+
+  it('wizard goes to previous step', () => {
+    const { result } = renderHook(() => useWizard({ initialKey: 4, steps }));
+
+    act(() => {
+      result.current[1].previous();
+    });
```

Comment:
one nitpicky thing - the naming of tests (look into this comment) https://github.com/openboxes/openboxes/pull/3701#discussion_r1045926241

Reply:
Where is that convention documented? I can't any resource opting for the "should" convention.

Reply:
do we have any conventions documented? :laughing: :laughing: :laughing: :laughing:

Reply:
When Alan was implementing tests a while ago, this was a request from me, _"It should..."_ is a very common pattern for test name that explains the behavior and expected result like
"should return initial key"
"should advance to the next step when next is called"
"should go to the previous step when previous is called'"
"should go to the specific step when set is called"